website/docs/cloud-docs/integrations/kubernetes/api-reference.mdx

```diff
@@ -0,0 +1,593 @@
+---
+page_title: Terraform Cloud Operator for Kubernetes API reference
+description: >-
+  API reference for the Terraform Cloud Operator for Kubernetes
+---
+
+# API Reference
+
+## Packages
+- [app.terraform.io/v1alpha2](#appterraformiov1alpha2)
```

Comment:
Is the plan to have more packages in the future? Right now this "Packages" straight to the only package feels a bit confusing